CRM/Event/WorkflowMessage/ParticipantTrait.php

```diff
@@ -67,6 +67,10 @@ trait CRM_Event_WorkflowMessage_ParticipantTrait {
    */
   protected $participantContacts;
 
+  private function isCiviContributionEnabled(): bool {
```

Comment:
not blocking but `isCiviContributeEnabled()` feels slightly more correct

Reply:
ok renamed now